### PR TITLE
Page CreateIdea category selection

### PR DESCRIPTION
**Pull request by fisx**
Obsoletes #242.

### CODE REVIEW

**Comment by fisx** [comment]
This fixes a few issues with `custom.js`.  The issue it does not fix is this: the click handlers for the buttons triggers successfully the state change for the radios, but once the handler returns, somebody else comes along and resets the state of the radios.

Neither @fisx nor @andorp have any idea who that might be so far.  I think we need someone who knows js...

File: static/js/custom.js [hunk 1]
```diff
@@ -70,4 +69,8 @@ function removeClass(el, cl) {
 
 function addClass(el, cl) {
     if(el) el.className = el.className + " " + cl;
-}
\ No newline at end of file
+}
+
+function endsWith(str, suffix) {
```
Comment (andorp) [inline static/js/custom.js]: Please remove if it not used.

**Comment by fisx** [comment]
@andorp thanks, removed dead code.

@local-girl could you take a look?  thanks!

**Comment by fisx** [comment]
Fixed and tested!  I commented out one test because I have other more urgent tasks (last commit).  If the reviewer has an idea how to fix it, I'd be interested.  Otherwise we can fix it later.

**Comment by andorp** [comment]
LGTM

**Pull request merged**